oci/private/push.bzl

```diff
@@ -180,6 +180,10 @@ _attrs = {
         """,
         allow_single_file = True,
     ),
+    "retry_count": attr.int(
```

Comment:
AFAIK, crane already has a retry built into the wire which handles thins like network jitter, and retryable http statuses.  

What's the reason for this addition?  

Also, you can easily do this in your shell calling into the `bazel run :push`.  Why not do that?